package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "js-wheatherapp",
+  "version": "1.0.0",
+  "description": "Microverse Fifth JS Project",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Dandush03/JS-WheatherApp.git"
+  },
+  "author": "Daniel Laloush",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Dandush03/JS-WheatherApp/issues"
+  },
+  "homepage": "https://github.com/Dandush03/JS-WheatherApp#readme",
+  "devDependencies": {
+    "css-loader": "^3.5.2",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "sass": "^1.26.3",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.4",
+    "webpack": "^4.42.1",
+    "wepack-cli": "0.0.1-security"
```

Comment:
The `webpack-cli` dependency is missing. A wrong dependency was added instead. We need to remove it and install the right one.
First:
`npm uninstall wepack-cli`
And then:
`npm uninstall webpack-cli`